eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -0,0 +1,25 @@
+<Project>
+  
+  <PropertyGroup>
+    <!-- Used to silence the warning caused by the workaround for https://github.com/dotnet/runtime/issues/81382 -->
+    <SuppressGenerateILCompilerExplicitPackageReferenceWarning>true</SuppressGenerateILCompilerExplicitPackageReferenceWarning>
+  </PropertyGroup>
+
+  <!-- needed to reference a specific version of NetCoreApp. Workaround https://github.com/dotnet/runtime/issues/81382 -->
```

Comment:
According to a @sbomer comment on this issue this will be fixed (or has already been fixed) soon. Is the expectation that once it is we can remove this whole thing and just add a RuntimeFrameworkVersion property to the project.csproj template?

Reply:
That is the hope. However this is blocked because the ASP.NET and Windows.Desktop try to get restored with the same version, which they don't have matching versions on daily builds. So we need a better story here how to reference a daily build of dotnet/runtime. That's the hope that will be fixed with the referenced issue.